internal/driver/config.go

```diff
@@ -17,8 +17,8 @@ import (
 type CustomConfig struct {
```

Comment:
On line 20, 
// DefaultSecretName indicates the secret `path` to retrieve username and password from secret store.